app/models/user/UserActions.scala

```diff
@@ -0,0 +1,103 @@
+package models.user
+
+import java.sql.Timestamp
+
+import com.github.tminglei.slickpg.InetString
+import controllers.sugar.Requests.AuthRequest
+import db.ModelService
+import db.impl.UserActionLogTable
+import db.impl.model.OreModel
+import enumeratum.values.{IntEnum, IntEnumEntry}
+import enumeratum.{EnumEntry, Enum}
+import ore.StatTracker
+import ore.user.UserOwned
+
+case class UserActionLogEntry(override val id: Option[Int] = None,
+                              override val createdAt: Option[Timestamp] = None,
+                              private val _userId: Int,
```

Comment:
Any reason for hiding the values here if they are then exposed again later? The bytecode is AFAIK identical.

Reply:
did not touch this